src/main/resources/migrations/common/V2024.04.22__baseline_tables.sql

```diff
@@ -190,7 +190,8 @@ CREATE TABLE video_booking
     created_time              timestamp    NOT NULL,
     amended_by                varchar(100),
     amended_time              timestamp,
-    migrated_video_booking_id bigint
+    migrated_video_booking_id bigint,
+    migrated_description      varchar(50)
```

Comment:
This should be big enough.  It is the same size as the old service [here](https://github.com/ministryofjustice/whereabouts-api/blob/main/src/main/resources/db/migration/V24__normalise_video_link_booking.sql#L8).